### PR TITLE
removing data from test PB article

### PR DESCRIPTION
**Pull request by ShahafBenYakir**

### CODE REVIEW

File: docs/integrations/test-playbooks.md [hunk 1]
```diff
@@ -3,11 +3,13 @@ id: test-playbooks
...
 
-We use Test Playbooks to test our integrations and automation scripts. The Test Playbooks provide full *End to End* testing. For testing small units of code, use [Unit Testing](unit-testing). 
 
-Tests are run using our CI framework. They are run both as part of the build process and on a nightly basis. If you are a contributor, during the initial PR, Test Playbooks will not run but they will be used as part of the review process. Once the PR is merged into a `contrib/*` branch by one of the team members and credentials (if needed) are provided to Cortex XSOAR, the test playbooks will be run as part of our CI framework. 
+We use Test Playbooks to test our integrations and automation scripts. The Test Playbooks provide full *End to End* testing. For testing small units of code, use [Unit Testing](unit-testing). 
+Tests are run using our CI framework. They are run both as part of the build process and on a nightly basis. 
 
-**Note:** Test Playbooks are required as part of the PR acceptance review process. For simple Scripts that have unit tests, a test playbook is optional.
+:::note
+The content team will automatically run Test Playbooks for packs that are not supported by XSOAR
```
Comment (Itay4) [inline docs/integrations/test-playbooks.md:11]: do we do that ?

i dont think this note is needed

File: docs/integrations/test-playbooks.md [hunk 2]
```diff
@@ -99,27 +101,6 @@ version: -1
 name: IPInfo-Test
 ```
 
-## Adding Tests to conf.json
-In order to associate integrations with a test plabyook we mange a `conf.json` file (at the root of the repository). The conf.json file is located in the **Tests** directory.
```
Comment (Itay4) [inline docs/integrations/test-playbooks.md:103]: i wouldnt remove it, as in some cases it is still needed

instead, i would write that by default it is not required to add to `conf.json`, but if needed then use as follows (and keep the rest)

Comment (ShahafBenYakir) [reply docs/integrations/test-playbooks.md:103]: If I add this back then definitely I would add the top 👆 note.

**Comment by ShahafBenYakir** [comment]
@Itay4 see latest commit

File: docs/integrations/test-playbooks.md [hunk 3]
```diff
@@ -3,11 +3,13 @@ id: test-playbooks
...
-We use Test Playbooks to test our integrations and automation scripts. The Test Playbooks provide full *End to End* testing. For testing small units of code, use [Unit Testing](unit-testing). 
 
-Tests are run using our CI framework. They are run both as part of the build process and on a nightly basis. If you are a contributor, during the initial PR, Test Playbooks will not run but they will be used as part of the review process. Once the PR is merged into a `contrib/*` branch by one of the team members and credentials (if needed) are provided to Cortex XSOAR, the test playbooks will be run as part of our CI framework. 
+We use Test Playbooks to test our integrations and automation scripts. The Test Playbooks provide full *End to End* testing. For testing small units of code, use [Unit Testing](unit-testing). 
+Tests are run using our CI framework. They are run both as part of the build process and on a nightly basis. 
 
-**Note:** Test Playbooks are required as part of the PR acceptance review process. For simple Scripts that have unit tests, a test playbook is optional.
+:::note
+By default the content team will not run Test Playbooks for packs that are not supported by XSOAR.
```
Comment (Itay4) [inline docs/integrations/test-playbooks.md:11]:
```suggestion
By default, Test Playbooks will not run in the CI for packs that are not supported by Cortex XSOAR.
```

File: docs/integrations/test-playbooks.md [hunk 4]
```diff
@@ -3,11 +3,12 @@ id: test-playbooks
 title: Test Playbooks
 ---
 
-We use Test Playbooks to test our integrations and automation scripts. The Test Playbooks provide full *End to End* testing. For testing small units of code, use [Unit Testing](unit-testing). 
-
-Tests are run using our CI framework. They are run both as part of the build process and on a nightly basis. If you are a contributor, during the initial PR, Test Playbooks will not run but they will be used as part of the review process. Once the PR is merged into a `contrib/*` branch by one of the team members and credentials (if needed) are provided to Cortex XSOAR, the test playbooks will be run as part of our CI framework. 
+:::note
+By default, Test Playbooks will not run in the CI for packs that are not supported by Cortex XSOAR. Hence, adding a test playbook as described in the following article is not required, unless specifically requested by XSOAR.
```
Comment (Itay4) [inline docs/integrations/test-playbooks.md:7]:
```suggestion
By default, Test Playbooks will not run in the CI for packs that are not supported by Cortex XSOAR. Hence, adding a test playbook as described in this article is not required, unless specifically requested by Cortex XSOAR.
```

**Review by Itay4 — APPROVED** [review]

File: docs/integrations/test-playbooks.md (hunk 4 above)
Comment (ShahafBenYakir) [reply docs/integrations/test-playbooks.md:7]: ishtabach

**Pull request merged**